ml-agents/tests/trainers/test_ppo.py

```diff
@@ -42,15 +42,15 @@ def dummy_config():
 
 @mock.patch('mlagents.envs.UnityEnvironment.executable_launcher')
 @mock.patch('mlagents.envs.UnityEnvironment.get_communicator')
-def test_ppo_policy_evaluate(mock_communicator, mock_launcher):
+def test_ppo_policy_evaluate(mock_communicator, mock_launcher, dummy_config):
```

Comment:
Has this been tested on both Windows and Mac ?

Reply:
Yes.